NachoNight/frontend
# Stylelint

Issue #7 

TL;DR - What does this implement?
Adds linting to css via stylelint

How to Test
Make sure errors pop up in vscode and that you have the stylelint plugin

Worries / Concerns
------------------

Checklist
---------

 - [x] Is this a code change?
 
ESLint Output
-------------

```
// Paste here
```

Reviewers
---------

(Tag people here)
@NachoNight/frontend
